git_machete/cmd.py

```diff
@@ -3415,8 +3416,8 @@ def parse_direction(b: str, allow_current: bool, down_pick_mode: bool) -> Option
             read_definition_file(verify_branches=False)
             b = opt_branch or current_branch()
             expect_in_managed_branches(b)
-            if params:
-                annotate(b, params)
+            if len(params) == 1:
+                annotate(b, params[0])
```

Comment:
Whoops... this is almost surely an error, `annotate` accepts a list of strings (see line 607: `annotations[b] = " ".join(words)`)

Reply:
Generally, would be nice if you eyeballed all other places where the changes included more than just adding type annotations 🤔 (i.e. the places where the logic has potentially changed)